source/deployment/ldap-group-sync.rst

```diff
@@ -0,0 +1,144 @@
+
+Teams and channels membership synchronization
+----------------------------------------------
+
+Default teams and channels will be added to the group members user interface when they login for the first time after the group has been configured with the specified teams and channels. It may take a few seconds to load all team and channel memberships for a user depending on the number of teams and channels the group is defaulted to.  This process is independent of the AD/LDAP synchronization process to support future functionality of groups. 
+
+.. note::
+Users are not removed from the team or channel on subsequent synchronizations of the AD/LDAP groups.  Users will need to be manually removed from the Team or Channel per the existing functionality.  They will not be re-added if they were manually removed.
+
+.. image:: ../images/Team_Channel_Membership_Sync.png
+
+Remove configured teams and channels from a group
+-------------------------------------------------
+To remove a team or channel configured for a group, click “Remove” to the right of the team or channel name. 
+
+View users belonging to the group
+---------------------------------
+
+Users who have logged in and accessed Mattermost will be visible in the members list on the group object. Members are read-only at this time and new members can be added through management in your AD/LDAP system. 
+
+.. image:: ../images/Group_Members.png
+
+Users will be removed from the group on subsequent synchronizations. However, they will not be removed from the teams and channels that they were defaulted to in the synchronization. If a user is removed from a AD/LDAP group and then later re-added, they will be defaulted again into the teams and channels configured in the group. If a member is removed altogether from the AD/LDAP system, all their team and channel memberships will be removed. 
+
+..note:: 
+When a member removes themselves manually from a channel, that action is tracked in the Channel Member History table.  Users are not re-added to channels from which they previously removed themselves. 
+
+Managing groups
+---------------
+Once a group has been configured, it can be edited to change the default team and channels by clicking “Edit” on the group on the list view.  
+
+Deleting groups
+---------------
+Groups can be deleted by adjusting your AD/LDAP group filter to remove the group or by unlinking the group on the groups listing page. If you add the group back by re-adjusting the AD/LDAP group filter and link the group again on the group configuration page, the previous configurations will be available.
+
+Frequently Asked Questions
+----------------------------
+
+Why is AD/LDAP Group Sync in alpha?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We want to make sure we have tested this feature in environments that have different AD/LDAP system and group structures to ensure the feature works well in the different varieties of environments our clients have. 
```

Comment:
What have been tested thus far?

Reply:
Should "clients" be "customers"?

Reply:
Updated clients and added the environments we have tested.